gammapy/datasets/io.py

```diff
@@ -191,12 +195,14 @@ def write_arf(self, dataset, filename):
 
         """
         aeff = dataset.exposure / dataset.exposure.meta["livetime"]
-        aeff.write(
-            filename=filename,
-            overwrite=self.overwrite,
-            format=self.format.replace("ogip", "ogip-arf"),
-            checksum=self.checksum,
-        )
+
+        filename = make_path(filename)
+        hdulist = aeff.to_hdulist(format=self.format.replace("ogip", "ogip-arf"))
```

Comment:
I do not really understand this replacement. Even if there is an ARF spec in OGIP (see https://heasarc.gsfc.nasa.gov/docs/heasarc/caldb/docs/memos/cal_gen_92_019/cal_gen_92_019.html), one can see there that:

> HDUCLASS = 'OGIP'
> HDUDOC = 'CAL/GEN/92-019'
> HDUCLASn
> HDUCLAS1 = 'RESPONSE'
> HDUCLAS2 = 'EFF_AREA'
> HDUVERS = '1.0.0'
> EXTNAME = 'EFFECTIVE AREA'  # (suggested, not required)

So, I am not sure that one needs a new "ogip-arf" format, also because the aeff code has not changed.

Can you precise the idea behind this?

Reply:
PS: the l360 of spectrum has not been updated accordingly...

Reply:
`ogip-arf` is an internal keyword for `RegionNDMap.to_hdulist` it is not a new format. 

In general, the handling of formats passed down to `Map.to_hdulist` is not very clear and would require cleanup. But that would be for another PR(s).